### PR TITLE
Created the basic text-image container

### PR DESCRIPTION
**Pull request by PratikGarai**
The container having image and text side by side + responsive css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,6 +64,36 @@
 
     <!--Content Starts-->
 
+	<main>
```
Comment (manakpandey) [inline index.html]: remove ```main``` tag

File: index.html [hunk 2]
```diff
@@ -63,6 +64,36 @@
...
+		<div class="site-section">
+			<div class="container">
+				<div class="half d-md-flex d-block align-items-stretch">
```
Comment (manakpandey) [inline index.html]: Try using the grid in bootstrap, instead of writing css yourself.

**Review by manakpandey — CHANGES_REQUESTED** [review]
Try to make use of bootstrap as much as you can.

**Comment by PratikGarai** [comment]
Made the changes. Also added an hr tag to enhance readability in responsive mode when the width is reduced.
image
As the width reduces, the alternate occurring text fields appear together.

File: index.html [hunk 3]
```diff
@@ -62,6 +63,32 @@
...
+			<a href="#" class="img-link"><img class="img" src="images/1.jpeg"></img></a>
+		</div>
+		<div class = "col-lg-6 col-md-12 d-md-flex d-block align-items-center">
+			<div class="content">
+				<h3>Some heading</h3>
+				<p> Some content . Image on left. Some really really long text so that the right hand side of the border can be verified.</p>
+			</div>
+		</div>
+	</div>	
+	<hr>
```
Comment (manakpandey) [inline index.html]: remove ```hr``` and add margin instead

**Review by manakpandey — CHANGES_REQUESTED** [review]

**Comment by PratikGarai** [comment]
Added the footer's basic structure. Icons for social media left.

**Pull request closed without merging**